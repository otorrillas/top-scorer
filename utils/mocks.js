export const mockRuns = [
  {
    videos: { links: [{ uri: 'https://video-url' }] },
    players: [{ id: 'playerId' }],
    times: { primary_t: 450 }
  }
];

export const mockGames = {
  '1234': {
    abbreviation: 'test',
    assets: {
      'cover-medium': {
        uri: 'https://cover-medium-url'
      }
    },
    runs: mockRuns
  }
};

export const mockGamesResponse = [
  {
    id: '1234',
    ...mockGames['1234']
  },
  {
    id: '45678',
    ...mockGames['1234']
  }
];
