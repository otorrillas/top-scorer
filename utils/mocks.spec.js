import React from 'react';

import { mockGames } from './mocks';

describe('Utils - mocks', () => {
  test('games', () => {
    expect(mockGames).toMatchSnapshot();
  });
});
