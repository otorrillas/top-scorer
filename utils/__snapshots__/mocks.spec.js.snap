// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils - mocks games 1`] = `
Object {
  "1234": Object {
    "abbreviation": "test",
    "assets": Object {
      "cover-medium": Object {
        "uri": "https://cover-medium-url",
      },
    },
    "runs": Array [
      Object {
        "players": Array [
          Object {
            "id": "playerId",
          },
        ],
        "times": Object {
          "primary_t": 450,
        },
        "videos": Object {
          "links": Array [
            Object {
              "uri": "https://video-url",
            },
          ],
        },
      },
    ],
  },
}
`;
